CHANGELOG.md

```diff
@@ -1,19 +1,56 @@
 # Changelog
 
+## Version 2.2.3 - release March 25, 2021
+
+### Bug Fixes
```

Comment:
I argued elsewhere, and received at least some agreement, that `Features/Improvements` should come before `Bug Fixes`. Seems pretty obvious to me, but.... 😉

Reply:
Sounds good to me!  Fixed.